code/api/src/migrations/8-user-add-description.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('users', 'description', {
+      allowNull: true,
+      type: Sequelize.TEXT
```

Comment:
user description is optional field

Reply:
Awesome! Have we looked into adding a test for creating a user? Can it create without these columns? What errors do we get on 'required' columns? Etc.

Reply:
@priyapower I haven't tried out a test for creating a user yet but that would be a good thing to test! Maybe we can start adding tickets for the legacy code to get tests on.